### PR TITLE
chore(database): Configure application to use Postgresql

### PR DESCRIPTION
**Pull request by kayroy247**
#### What does this PR do?
This PR configures Author's Haven to use Postgres as the database engine
#### Description of Task to be completed?
This task will be completed with the creation of databases on Heroku and using the connection URI in the connection string
#### How should this be manually tested?
This can be mannually tested by starting the app
#### Any background context you want to provide?
Author's Haven comes with the support for MongoDB but is now configured to use Postgresql
#### What are the relevant pivotal tracker stories?
#161780551
### Screenshots (if appropriate)
image

### CODE REVIEW

File: .env.sample [hunk 1]
```diff
@@ -0,0 +1,2 @@
+DEV_DATABASE_URL=
+DATABASE_URL=
```
Comment (fuchodeveloper) [inline .env.sample]: Add end of line

File: index.js [hunk 2]
```diff
@@ -39,18 +43,26 @@ if (!isProduction) {
...
-    mongoose.connect(process.env.MONGODB_URI);
+    const sequelize = new Sequelize(process.env.DATABASE_URL);
 } else {
-    mongoose.connect("mongodb://localhost/conduit");
-    mongoose.set("debug", true);
+    const sequelize = new Sequelize(process.env.DEV_DATABASE_URL);
+    sequelize
+        .authenticate()
+        .then(() => {
+            console.log('Database connection successfull.');
```
Comment (fuchodeveloper) [inline index.js]: Remove console log statements

File: .env.sample (hunk 1 above)
Comment (kayroy247) [reply .env.sample]: Ok thanks. I will add it.

File: index.js (hunk 2 above)
Comment (kayroy247) [reply index.js]: Okay. I will do that.

File: src/index.js [hunk 3]
```diff
@@ -1,93 +1,81 @@
...
-        res.json({
-            errors: {
-                message: err.message,
-                error: err
-            }
-        });
+  // eslint-disable-next-line no-unused-vars
+  app.use((err, req, res, next) => {
+    // eslint-disable-next-line no-console
+    console.log(err.stack);
```
Comment (fuchodeveloper) [inline src/index.js]: No console log statements in your pull requests

**Review by fuchodeveloper — CHANGES_REQUESTED** [review]

File: src/index.js (hunk 3 above)
Comment (fuchodeveloper) [reply src/index.js]: Remove log statements

**Review by fuchodeveloper — CHANGES_REQUESTED** [review]
Resolve merge conflicts

**Review by fuchodeveloper — COMMENTED** [review]
gitignore the `.DS_Store` file. It should not be added to the repo

**Comment by Joyce-O** [comment]
LGTM

**Review by fuchodeveloper — APPROVED** [review]

**Pull request merged**